src/components/ImageCard/ImageCard.tsx

```diff
@@ -62,15 +62,23 @@ export const ImageCard = ({
       </Link>
       <div className="cc-image-card__body">
         <span className="cc-image-card__meta">
-          <span className="cc-image-card__meta-item cc-image-card__meta-item--flag cc-image-card__meta-item--topic">
+          <span className="cc-image-card__meta-item cc-image-card__meta-item--type cc-image-card__meta-item--flag">
             {topic}
           </span>
-          <span
-            className="cc-image-card__meta-item cc-image-card__meta-item--author"
-            itemProp="author"
-          >
-            {author}
-          </span>
+          {authors && (
+            <dl className="cc-image-card__meta-item cc-image-card__meta-item--author cc-image-card__authors">
+              <dt className="cc-image-card__authors-label">Author</dt>
```

Comment:
I am wondering if it should change to `Authors` when more than one author. (I haven’t seen the design for this, so please ignore if this is not relevant)

Reply:
Ah it's interesting you raised that... it's not relevant in the way you think it might be...

The designs for these are simply an author's name (e.g. below), but it's interesting that you thought of the Author component, and makes me think maybe this prop would be better declared as `authorNames`...

Reply:
I have a job to consolidate some of the code for the new Card components, and the author names are potentially going to be extracted a new component so I will bare this in mind!